robots/ur_description/urdf/ur10_joint_limited_robot.urdf

```diff
@@ -32,15 +32,15 @@
   <link name="base_link">
     <visual>
       <geometry>
-        <mesh filename="package://ur_description/meshes/ur10/visual/base.dae"/>
+        <mesh filename="package://example-robot-data/robots/ur_description/meshes/ur10/visual/base.dae"/>
```

Comment:
I already think about that, and changed my mind: https://gepgitlab.laas.fr/gepetto/example-robot-data/merge_requests/29/diffs

Maybe we should discuss a bit about that.

Reply:
Yes, this will impact everybody (Pinocchio, crocodyl). I think we should move back, otherwise the policy is not the same for all the robots inside example-robot-data!

Reply:
@nim65s Do you have in mind to revert it?

Reply:
I think we should revert it, yes. 
@florent-lamiraux @jmirabel do you want to add anything ?

Reply:
It is an accurate change in the meaning of the package - e.g. meshes being found in RViz (path relative to the package.xml). I think these changes could make sense. **However, I think we should have more discussion and review before quickly merging and (potentially) breaking such a widely used package - especially with such a generic title.**  @florent-lamiraux @nim65s @jcarpent

Reply:
When we started the repo, we aimed to do not change URDF files compared with its main sources. The idea was to quickly update example-robot-data sources once we have new changes in the source files.

If we starte to modify URDF files, then we will lose this trackability, and it's a CON. Probably we could figure out a different solution to this PR.

Reply:
When we started the repo, we aimed to do not change URDF files compared with its main sources. The idea was to quickly update example-robot-data sources once we have new changes in the source files.

If we starte to modify URDF files, then we will lose this trackability, and it's a CON. Probably we could figure out a different solution to this PR.